client/ayon_harmony/plugins/create/create_farm_render.py

```diff
@@ -4,17 +4,15 @@
 from ayon_harmony.api import plugin
 
 
-class CreateFarmRender(plugin.Creator):
+class CreateFarmRender(plugin.HarmonyCreator):
```

Comment:
BTW why this has separate create plugin?

Reply:
Removed superfluous creator.